Complete fizz_buzz assignment

RyanSpittler opened the pull request:
Finished the assignment, fizz_buzz.rb from the presentation,
methods_and_flow_control.

Pivotal Tracker Story: [#88483086]

RyanSpittler committed:
Complete fizz_buzz assignment

Finished the assignment, fizz_buzz.rb from the presentation,
methods_and_flow_control.

Pivotal Tracker Story: [#88483086]

jasonnoble commented on methods_and_flow_control/fizz_buzz.rb:
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+# Adding these methods to Integers makes the whole read easier.
```
Yes, that may be true, but monkey patching is a bad idea...

jasonnoble commented on methods_and_flow_control/fizz_buzz.rb:
```diff
@@ -0,0 +1,22 @@
+# Adding these methods to Integers makes the whole read easier.
+class Fixnum
+  def divisible_by?(divisor)
+    if self % divisor == 0
```
`self % divisor == 0` is all you need for this method.  That line will return true/false for you.

jasonnoble commented on methods_and_flow_control/fizz_buzz.rb:
```diff
@@ -0,0 +1,22 @@
+  end
+
+  def fizzbuzzer
+    if self.divisible_by?(3) || self.divisible_by?(5)
```
Very clever, but is it easy to comprehend? :)

jasonnoble commented:
Needs rebasing.

RyanSpittler committed:
Remove class defined methods

- Removed divisible_by? method.
- Removed the fizzbuzzer from the number class

[#88483086]
